src/helpers/proofs/getProofByWay.ts

```diff
@@ -8,7 +8,12 @@ import scheduleProofJob from 'helpers/proofs/scheduleProofJob'
 const getProofByWay = {
   centralized: async (proverAddress?: string) => {
     if (!proverAddress) throw new Error(ErrorType.prover)
-
+    if (
+      !proverAddress.startsWith('https://') &&
+      !proverAddress.startsWith('http://')
```

Comment:
We can do simpler:
`!proverAddress.startsWith('http')`
+ you don't need another check for `https`